.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
   license-check:
     docker:
-      - image: circleci/ruby:2.5
+      - image: circleci/ruby:2.6
```

Comment:
@valerymo isn't this the same as the one we have merged in a separate PR? If so, please remove this changes.

Reply:
@MStokluska , it's some technical issue. I removed and tried rebase, and it's not removed. The reason maybe becasue  this actually was done first, before separate PR. Can we merge it as is?

Reply:
@MStokluska I rebased once more. And it looks ok now - circleci/config - disapered here.
Thank you